modules/codegen/src/main/scala/com/twilio/guardrail/ClientGenerator.scala

```diff
@@ -34,28 +34,26 @@ object ClientGenerator {
   )(implicit C: ClientTerms[L, F], Fw: FrameworkTerms[L, F], Sc: LanguageTerms[L, F], Sw: SwaggerTerms[L, F]): F[Clients[L]] = {
     import C._
     import Sw._
+    import Sc._
     for {
       clientImports      <- getImports(context.tracing)
       clientExtraImports <- getExtraImports(context.tracing)
       supportDefinitions <- generateSupportDefinitions(context.tracing, securitySchemes)
       clients <- groupedRoutes.traverse({
         case (className, unsortedRoutes) =>
-          val routes     = unsortedRoutes.sortBy(r => (r.path.unwrapTracker, r.method))
-          val clientName = s"${className.lastOption.getOrElse("").capitalize}Client"
-          def splitOperationParts(operationId: String): (List[String], String) = {
-            val parts = operationId.split('.')
-            (parts.drop(1).toList, parts.last)
-          }
-
+          val routes = unsortedRoutes.sortBy(r => (r.path.unwrapTracker, r.method))
           for {
+            clientName <- formatTypeName(className.lastOption.getOrElse(""), Some("Client"))
```

Comment:
I think this is a great opportunity to push the string `"Client"` into each generator, further reducing the impedance mismatch when moving to support non-pascal-case languages.

Reply:
This is fine, actually, and i even could have made the param value "client" or "CLIENT" or whatever; the formatter is responsible for doing the right thing for the language.  I wanted the generators to _not_ be responsible for naming things, as much as is reasonably possible.

Reply:
makes sense, though hopefully my reticence towards hard-coded strings in core is understandable